app/ab_test/a/templates/Donation_Confirmation_Phone.html.twig

```diff
@@ -116,7 +116,7 @@
 						</div>
 
 						<div class="payment f-left">
-							{% if donation.paymentType == 'PPL' or donation.paymentType == 'MCP' %}
+							{% if donation.paymentType == 'PPL' or donation.paymentType == 'MCP' or donation.paymentType == 'SUB' %}
```

Comment:
@KaiNissen You mean this is, given the lack of detailed status, more a case for the text simply saying "Sie zahlen per SUB" (cp. BEZ)?